Added Functionality of Thank-You mail

nishantKathrotiya opened the pull request:
In this pull request, I implemented a robust thank-you email system for contributors who have actively engaged with our projects. The following key features were added:

### **1. script/thank-you-mail.js**

- I created a script that efficiently fetches the email addresses of all contributors who have merged at least one pull request (PR). This targeted approach ensures that we recognize and appreciate those who have made significant contributions to our community. After retrieving the relevant emails, the script calls a function to send the thank-you emails.

### **2. Updated utils/mailer.js**

- I enhanced the mailer utility by adding a new function that loads the EJS template for the thank-you email. This function constructs the email body and triggers the sending process, ensuring a seamless integration with our existing mailing infrastructure. By streamlining this process, we improve our ability to maintain clear and consistent communication with contributors.

### **3. templates/thank-you.ejs**

- I developed a general-purpose EJS template for the email body. It serves as a foundation that can be easily customized for future email communication

closes #32

jalaym825 commented:
<img width="602" alt="image" src="https://github.com/user-attachments/assets/e1658512-a43c-4c5d-b4e3-23913ae4695c">

add logo in header

![WhatsApp Image 2024-10-27 at 19 10 41](https://github.com/user-attachments/assets/a2b291b3-62ae-4af8-8f9d-ff4768ad2bb5)

footer like this, but don't include my personal number 😅
even I didn't know that I was sending my number in mail in PR Merged template.
instead of twitter, add discord link

nishantKathrotiya commented:
Ok Brother i make sure i don't include your number

nishantKathrotiya commented:
![WhatsApp Image 2024-10-28 at 13 30 12_0d207be0](https://github.com/user-attachments/assets/27ce0b25-3584-4742-9b84-598ec005c254)

Updated Header and footer 
i have removed Name and Number in Footer and ignore here all the images are same in footer but in template it is fixed

jalaym825 commented:
remove charusat and done.

nishantKathrotiya commented:
I have removed Charusat From title and now it is "**_Hacktoberfest 2024_**"
